tests-legacy/Integration/classes/CartGetOrderTotalTest.php

```diff
@@ -453,8 +453,8 @@ public function testBasicRoundTypeLine()
         $cart->updateQty(1, $product_a->id);
         $cart->updateQty(1, $product_b->id);
 
-        $this->assertEquals(3.59, $cart->getOrderTotal(false, Cart::ONLY_PRODUCTS));
-        $this->assertEquals(4.29, $cart->getOrderTotal(true, Cart::ONLY_PRODUCTS));
+        $this->assertEquals(3.581, $cart->getOrderTotal(false, Cart::ONLY_PRODUCTS));
```

Comment:
Since the precision will likely be higher (3 * 2 quite often) Maybe you should increase the number of decimals to 6 digits? (At least in one product)

Reply:
You mean having a test where the end result has 6 digits? That's a good idea, I'll do that 👍